midterm-2023-11-07/code/5037-stefani-georgievska/script.ts

```diff
@@ -0,0 +1,137 @@
+interface Movies {
```

Comment:
The interface is more in line with what will be shown on the screen, not with what we're getting from the data. +5